src/DGtal/geometry/curves/Naive3DDSSComputer.ih

```diff
@@ -0,0 +1,587 @@
+{
+    blockXY = other.blockXY;
+    blockXZ = other.blockXZ;
+    blockYZ = other.blockYZ;
+    lenXY = other.lenXY;
+    lenXZ = other.lenXZ;
+    lenYZ = other.lenYZ;
+}
+
+/**
+ * Assignment.
+ * @param other the object to copy.
+ * @return a reference on 'this'.
+ * Forbidden by default.
+ */
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity> &
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::operator= (
+                                      const Naive3DDSSComputer<TIterator,TInteger,connectivity> & other )
+{
+  myProjXY = other.myProjXY;
+  myProjXZ = other.myProjXZ;
+  myProjYZ = other.myProjYZ;
+  myXYalgo = other.myXYalgo;
+  myXZalgo = other.myXZalgo;
+  myYZalgo = other.myYZalgo;
+  myBegin = other.myBegin;
+  myEnd = other.myEnd;
+  blockXY = other.blockXY;
+  blockXZ = other.blockXZ;
+  blockYZ = other.blockYZ;
+  lenXY = other.lenXY;
+  lenXZ = other.lenXZ;
+  lenYZ = other.lenYZ;
+  return *this;
+}
+
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+typename DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::Self
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::getSelf() const {
+  return Self();
+}
+
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+typename DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::Reverse
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::getReverse() const {
+  return Reverse();
+}
+
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+bool
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::isInDSS ( const Point3d & point) const {
+  short test = 0;
+  if ( myXYalgo.isInDSS ( myProjXY ( point ) ) )
+      test++;
+  if ( myXZalgo.isInDSS ( myProjXZ ( point ) ) )
+      test++;
+  if ( myYZalgo.isInDSS ( myProjYZ ( point ) ) )
+      test++;
+
+  return test >= 2 ? true : false;
+}
+
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+bool
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::isInDSS ( const ConstIterator & it) const {
+  short test = 0;
+  if ( myXYalgo.isInDSS ( myProjXY ( *it ) ) )
+     test++;
+  if ( myXZalgo.isInDSS ( myProjXZ ( *it ) ) )
+      test++;
+  if ( myYZalgo.isInDSS ( myProjYZ ( *it ) ) )
+      test++;
+
+  return test >= 2 ? true : false;
+}
+
+/**
+ * Equality operator.
+ * @param other the object to compare with.
+ * @return 'true' either if the points perfectly match
+ * or if the first points match to the last ones
+ * (same DSS scanned in the conversed way)
+ * and 'false' otherwise
+ */
+template <typename TIterator, typename TInteger, int connectivity>
+inline
+bool
+DGtal::Naive3DDSSComputer<TIterator,TInteger,connectivity>::operator==(
+                                      const Naive3DDSSComputer<TIterator,TInteger,connectivity>& other ) const
+{
+  return (  (	myXYalgo == other.myXYalgo ) &&
```

Comment:
The code does not seem to match the comment (about DSS scanned in conversed way). Seems there is only one test here.